packages/rest-typings/src/v1/users.ts

```diff
@@ -110,6 +112,11 @@ export type UserPresence = Readonly<
 
 export type UserPersonalTokens = Pick<IPersonalAccessToken, 'name' | 'lastTokenPart' | 'bypassTwoFactor'> & { createdAt: string };
 
+export type PickedUser = Pick<
```

Comment:
```suggestion
export type PickedUser = Pick<
```
Good catch on finding the missing properties here. Can we maybe define it with a more descriptive name? Maybe `UserWithDefaultInfo` or `UserListInfo`